ReportPortal.Tests/ReportPortal.E2E.Core/HttpMessageHandlers/ClientsHandler.cs

```diff
@@ -1,54 +1,48 @@
 ﻿using System.Collections.Concurrent;
-using System.Net.Http.Json;
 using Microsoft.Extensions.DependencyInjection;
+using ReportPortal.E2E.Core.HttpMessageHandlers.HttpClient;
 using ReportPortal.E2E.Core.Models;
 
 namespace ReportPortal.E2E.Core.HttpMessageHandlers
 {
     public class ClientsHandler
     {
         private static ConcurrentDictionary<string, AuthorizationMessageHandler> _authenticationHandlers;
-        private readonly HttpClient _httpClient;
+        private readonly IHttpClient _httpClient;
+        private readonly string _baseUrl = TestsBootstrap.Instance.ServiceProvider.GetRequiredService<ReportPortalConfig>().BaseUrl;
 
         public ClientsHandler()
         {
             _authenticationHandlers = new ConcurrentDictionary<string, AuthorizationMessageHandler>();
-            _httpClient = new HttpClient();
+            //_httpClient = new HttpClient.HttpClient(_baseUrl);
```

Comment:
Propose set client base of flag from config

Reply:
Done